canonical-web-and-design/juju.is
# Build the about mission page

## QA
- Go to http://0.0.0.0:8041/about/mission
- Check the content matches [the copy doc](https://docs.google.com/document/d/15Xnp-hjoMYguvqlrGzv-sAWw8GlbGfC7MBAJedWTIYU/edit)
- Design might want to check this one over 

Fixes https://github.com/canonical-web-and-design/juju.is/issues/97
